### PR TITLE
SVCS-2460 Added double encoding to the 8.0.0 changelog

### PR DESCRIPTION
**Pull request by chad-pelly**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [8.0.0]
+
+### Changed
```
Comment (AaronTrippaers) [inline CHANGELOG.md:10]: Should there a mention somewhere that these are breaking?

Comment (AaronTrippaers) [reply CHANGELOG.md:10]: We did this in earlier versions: https://github.com/ExtraHorizon/javascript-sdk/blob/ebbc3600e370d791457c4e98eba7a84ddee334e7/CHANGELOG.md#breaking-changes-1

Comment (chad-pelly) [reply CHANGELOG.md:10]: Hmmmm CHANGED is breaking:

These are the guidelines we state we follow: https://keepachangelog.com/en/1.1.0/

Comment (AaronTrippaers) [reply CHANGELOG.md:10]: Then we are using changed wrong in all the minor and patched version 😬 
I would rather be consistent with ourselves at this point then with the standard.

Comment (wwaaijer-exh) [reply CHANGELOG.md:10]: I'm leaning towards following the standard, but the standard also states it should be "painfully clear" that something is breaking 😄 Would it make sense to add a bold **BREAKING** in front of the item? In my opinion it should be more clear this is breaking, but no hard stance on how 🤔

**Review by wwaaijer-exh — COMMENTED** [review]
Also nice! 👌

File: CHANGELOG.md (hunk 1 above)
Comment (chad-pelly) [reply CHANGELOG.md:10]: We can meet in the middle for sure 🤗

**Review by AaronTrippaers — APPROVED** [review]

**Pull request merged**